source/module.c

```diff
@@ -22,12 +22,22 @@
 
 #include <aws/mqtt/mqtt.h>
 
+#if PY_MAJOR_VERSION == 3
+#    define INIT_FN PyInit__aws_crt_python
+#    define UNICODE_GET_BYTES_FN PyUnicode_DATA
```

Comment:
Do we want to use [PyUnicode_AsUTF8AndSize](https://docs.python.org/3/c-api/unicode.html#utf-8-codecs) so that we're sure we're getting utf-8?